### PR TITLE
FrontEnd: Search Engine Optimisation basics 

### LINKED ISSUE #933: FrontEnd: Search Engine Optimisation basics
<hr>

## Description 📜

Search engine optimisation basics documentation
<hr>

## Domain of Contribution 📊

- [x] Frontend Dev (HTML, CSS, JS)

<hr>

### PR DESCRIPTION
**Pull request by Yashesvinee**
<hr>

## Description 📜

Added Documentation of Search engine optimization
Fixes #933 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev (HTML, CSS, JS)

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Yashesvinee** [comment]
@praveenscience Sir

**Comment by Yashesvinee** [comment]
@2024-SANDHYA  could you please review this PR

**Comment by 2024-SANDHYA** [comment]
> 
> 
> @2024-SANDHYA could you please review this PR

LGTM !!

File: Web_Development/FrontEnd/SEO_Basics/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ - ### Google stores all its web pages in an ***index***. Every ***index*** entry has a Description and URL assiciated with it. *To index* is when Google fetches a page, reads it, and adds it to the index: *Google indexed several pages on my site today*.
+ - ### Google *crawls* the web, looking for new pages, then indexes them if necessary with the help of its *crawler*- the *GoogleBot*. GoogleBot is an automated software that acts as a librarian to help us find what we are searching for.<br>
+ ----
+### The techniques and factors which work in co-relation with each other to improve the overall SEO of your website are known as the SEO Success Factors. The Periodic Table of SEO Factors organizes the factors into six main categories and weights each based on its overall importance to SEO. In this section our main focus would be on the factors related to HTML Codes.<br>HTML is the coding standard used to make most web pages. Search engines can pick up specific signals from various HTML elements. So, the designers must use these HTML elements effectively to gain maximum advantage.<br><br>Here are some of the HTML elements to consider for SEO.
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/SEO_Basics/README.md:12]: `###` makes all content like a heading. Could use normal text at some areas like these.

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
Left a small review, do check.
All other things looks fine and well 😄💯

File: Web_Development/FrontEnd/SEO_Basics/README.md (hunk 1 above)
Comment (Yashesvinee) [reply Web_Development/FrontEnd/SEO_Basics/README.md]: I have made the changes 
is it okay now?

**Review by code-reaper08 — APPROVED** [review]
Looks fine now!

**Pull request merged**